feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature1 branch
+
```

Comment:
Why there is a space here? Can you explain?

Reply:
It is just to differentiate it